moveit2.repos

```diff
@@ -12,3 +12,7 @@ repositories:
     type: git
     url: https://github.com/moveit/moveit_resources.git
     version: ros2
+  ompl:
```

Comment:
Should we wait till the new ompl version is released?

Reply:
I guess the ompl crew would be willing to release although they still have [at least one unresolved regression](https://github.com/ompl/ompl/pull/1163) lying around.

But I'm not sure it's so difficult to support both ompl cmake interfaces here for now. [We do that in MoveIt](https://github.com/moveit/moveit/commit/cf575b02f65ee7741b18da5ab736f111de977796).